### PR TITLE
Add some safeguards for using low level crafting books.

### LINKED ISSUE #1994: Warn if you outlevel a crafting book
We could add a check for people with 175 skill and bail with a warning when using an apprentice book instead of the scripts hanging trying to craft something not in the book.

### PR DESCRIPTION
**Pull request by Sarvatt**
Closes #1994

### CODE REVIEW

File: remedy.lic [hunk 1]
```diff
@@ -67,6 +73,7 @@ class Remedy
     stow_item(left_hand)
     stow_item(right_hand)
     bput('get remed book', 'What were', 'You get')
+    echo('*** You will need to upgrade to a journeyman or master book before 176 ranks! ***') if DRSkills.getrank('Engineering') == 175
```
Comment (rpherbig) [inline remedy.lic]: ❓ 'Alchemy'?

File: sew.lic [hunk 2]
```diff
@@ -107,6 +107,7 @@ class Sew
 
   def study_tailoring(chapter, recipe)
     get_item('tailoring book')
+    echo('*** You will need to upgrade to a journeyman or master book before 176 ranks! ***') if DRSkills.getrank('Outfitting') == 175
     bput("turn my book to page #{find_recipe(chapter, recipe)}", 'You turn your', 'The book is already')
```
Comment (rpherbig) [inline sew.lic]: 💡 I like the exit-early idea. Would that be easier if `find_recipe` returned a bool?

**Comment by rpherbig** [comment]
❓ Could `smith.lic` get a check on looking up the section in the book?

**Comment by Sarvatt** [comment]
I'm not sure on that last commit https://github.com/rpherbig/dr-scripts/pull/3007/commits/7d271bcd26dbf79c4468ac61f49d212e9d38183b and want to test it, but it should work

**Comment by Sarvatt** [comment]
no luck there. hmm.

**Comment by Sarvatt** [comment]
got it
```[sew]>read my book
              -=    Chapter 5 : Knitted Clothing    =-

 Page 1: a knitted ankleband                     Page 10: some knitted booties
 Page 2: some knitted socks                      Page 11: a knitted hood
 Page 3: a knitted armband                       Page 12: a knitted towel
 Page 4: a knitted napkin                        Page 13: some knitted legwarmers
 Page 5: some knitted mittens                    Page 14: some knitted gloves
 Page 6: a knitted headband                      Page 15: a knitted shirt
 Page 7: some knitted slippers                   Page 16: a knitted skirt
 Page 8: a knitted scarf
 Page 9: a knitted hat
>
[sew: *** No match was found after 15 seconds, dumping info]
[sew: messages seen length: 11]
[sew: message: Gauge Flow  (29 roisaen)]
[sew: message:  Page 9: a knitted hat                           ]
[sew: message:  Page 8: a knitted scarf                         ]
[sew: message:  Page 7: some knitted slippers                   Page 16: a knitted skirt]
[sew: message:  Page 6: a knitted headband                      Page 15: a knitted shirt]
[sew: message:  Page 5: some knitted mittens                    Page 14: some knitted gloves]
[sew: message:  Page 4: a knitted napkin                        Page 13: some knitted legwarmers]
[sew: message:  Page 3: a knitted armband                       Page 12: a knitted towel]
[sew: message:  Page 2: some knitted socks                      Page 11: a knitted hood]
[sew: message:  Page 1: a knitted ankleband                     Page 10: some knitted booties]
[sew: message:               -=    Chapter 5 : Knitted Clothing    =-]
[sew: checked against [/Page \d+.*knitted hose/i]]
[sew: for command read my book]
[sew]>turn my book to page
Which page do you wish to turn the a book of apprentice tailoring instructions to?
>
[sew: ** Recipe not found in book, it needs to be upgraded to the next higher level ***]
[sew]>release spell
You let your concentration lapse and feel the spell's energies dissipate.
[sew]>release mana
You aren't harnessing any mana.
[sew]>release symb
You pause for a moment as the details of the Activate symbiosis fade from your mind.
You release the Activate symbiosis.```

**Pull request merged**